website/content/docs/release-notes/v0_16_0.mdx

```diff
@@ -312,6 +312,25 @@ description: >-
     </td>
   </tr>
 
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+    0.8.0+ (Fixed in Boundary Community Edition and Boundary Enterprise 0.16.4, 0.17.3, 0.18.2)
```

Comment:
What does 0.8.0+ mean here?

Reply:
Thanks for the review! It is intended to show that the issue was present in versions 0.8.0 and forward (which I got from the CVE ticket). It does look a little out of place here since it's such an early version. Maybe we should display that as `0.8.0 - 0.16.3` instead?

Reply:
Yeah that might be more easily understandable!